src/Flicking.js

```diff
@@ -1743,6 +1756,38 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
 		return this;
 	}
 
+	/**
+	 * Rebuild/Initialize panels by current DOM of panels.
+	 * @ko 현재 패널 DOM 을 기준으로 패널을 재구성/초기화한다.
+	 * @method eg.Flicking#rebuild
+	 * @example
+	 * 		flicking.rebuild();
+	 *
+	 * @return {eg.Flicking} An instance of a module itself <ko>모듈 자신의 인스턴스</ko>
+	 */
+	rebuild() {
+		const panel = this._conf.panel;
+		const options = this.options;
+		const prefix = options.prefix;
+
+		// filter original panel (remove clones)
+		utils.toArray(this.$wrapper.querySelectorAll(`.${prefix}-clone`)).forEach(el => el.remove());
```

Comment:
@jongmoon 

IE does not support 'remove' methods.

I think you should use parentNode.removeChild.

Reply:
Awesome!